man/os-release.xml

```diff
@@ -437,6 +437,20 @@
           </para></listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>SYSCFG_LEVEL=</varname></term>
+
+          <listitem><para>A lower-case string (mostly numeric, no spaces or other characters outside of 0–9,
+          a–z, ".", "_" and "-") identifying the operating system configuration support level, to indicate which
+          configuration images are supported. See <filename>/etc/syscfg-release.d/syscfg-release.<replaceable>IMAGE</replaceable></filename>,
+          <ulink url="https://docs.kernel.org/admin-guide/initrd.html">initrd</ulink> and
+          <citerefentry><refentrytitle>systemd-syscfg</refentrytitle><manvolnum>8</manvolnum></citerefentry>)
+          for more information.</para>
+
+          <para>Examples: <literal>SYSCFG_LEVEL=2</literal>, <literal>SYSCFG_LEVEL=15.14</literal>.
+          </para></listitem>
+        </varlistentry>
```

Comment:
don't copy the docs. just say "Samantically the same as SYSEXT_LEVEL=, just for syscfg images"